hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/bulk/TestRowDataKeyGen.java

```diff
@@ -191,6 +192,8 @@ void testPrimaryKeylessWrite() {
 
   @Test
   void testRecoredKeyContainsTimestamp() {
+    TimeZone timeZone = TimeZone.getTimeZone("GMT+8");
+    TimeZone.setDefault(timeZone);
     Configuration conf = TestConfigurations.getDefaultConf("path1");
```

Comment:
Don't think we should do this `TimeZone.setDefault(timeZone);`, the code affects the entire JVM process.

Reply:
GET.